Pod/Classes/WPAssetViewController.m

```diff
@@ -161,7 +171,8 @@ - (void)showError:(NSError *)error {
 - (void)handleTapOnAsset:(UIGestureRecognizer *)gestureRecognizer
 {
     if (gestureRecognizer.state == UIGestureRecognizerStateEnded) {
-        [self.navigationController setNavigationBarHidden:!self.navigationController.isNavigationBarHidden animated:YES];
+        [self.navigationController setNavigationBarHidden:!self.videoView.controlToolbarHidden animated:YES];
```

Comment:
I think we should probably also hide the status bar when we hide the navigation bar / control bar? Otherwise it's overlaid on the video and we don't get 'proper' fullscreen.